docs/configs/nav/nav_v0.2.yml

```diff
@@ -56,3 +56,6 @@ nav:
   - How to Deploy and Run with Helm?: onos-gui/docs/deployment.md
 - ONOS Library Packages:
     - Logging: onos-lib-go/docs/logging.md
+- TLA+ Conformance Monitoring Tools:
```

Comment:
Maybe we should just make this a `Logging & Monitoring` section? We might want to put some other docs that apply to all subsystems here, e.g. logging configuration.